Wrappers/Python/ccpi/web_viewer/trame_viewer.py

```diff
@@ -420,84 +252,42 @@ def construct_slice_window_range_slider(self):
             step=step,
             hide_details=True,
             dense=True,
-            disabled=self.disable_2d,
+            disabled=disabled,
             label="Slice window",
             thumb_label=True,
             style=style,
         )
 
-    def construct_color_slider(self):
-        if self.cmax > 100:
-            # Use actual values
-            min_value = self.cmin
-            max_value = self.cmax
-            step = 1
-            self.color_slider_is_percentage = False
-        else:
-            # Use percentages
-            min_value = 0
-            max_value = 100
-            step = 0.5
-            self.color_slider_is_percentage = True
-
-        return vuetify.VRangeSlider(label="Color range",
-                                    hide_details=True,
-                                    solo=True,
-                                    v_model=("coloring", self.windowing_defaults),
-                                    min=min_value,
-                                    max=max_value,
-                                    step=step,
-                                    disabled=self.disable_3d,
-                                    thumb_label=True,
-                                    style="max-width: 300px")
-
-    def construct_windowing_slider(self):
-        if self.cmax > 100:
-            # Use actual values
-            min_value = self.cmin
-            max_value = self.cmax
-            step = 1
-            self.windowing_slider_is_percentage = False
+    def update_slice_data(self):
+        if hasattr(self.cil_viewer, "getSliceMapWindow"):
+            self.cmin, self.cmax = self.cil_viewer.getSliceMapWindow((0., 100.))
+            self.slice_window_range_defaults = self.cil_viewer.getSliceMapWindow((60., 95.))
```

Comment:
why is this 60 and 95?

Reply:
This is close to the default values that are being used by the CILViewer2D I can find no reason why they are, I got to these numbers through trial and error and comparison with the (60. 95.) and default from the 2D viewer.

Reply:
That's weird, because the 2D viewer sets the percentiles here:
https://github.com/vais-ral/CILViewer/blob/47f45a970d6931d34173c5418737f765ab83658b/Wrappers/Python/ccpi/viewer/CILViewer2D.py#L1625

Reply:
Yeah I know, it's a very weird behaviour that I can't find the source for.

Reply:
what happens when you set 5, 95?

Reply:
problem could've been related to incorrect window definition in viewer?

Reply:
The min value was 0. up until about 60% but the viewer was setting it to about 130-150. which is roughly what we get when we use 60% as the min value.

Reply:
Part of the problem is the getSliceMapWindow method isn't getting the window for the slice (as previously discussed). So 5, 95 will be correct when the other issues are fixed.